main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-tools/go-steputils/stepconf"
+	shellquote "github.com/kballard/go-shellquote"
+)
+
+type config struct {
+	AdditionalParams string `env:"additional_params"`
+}
+
+func failf(msg string, args ...interface{}) {
+	log.Errorf(msg, args...)
+	os.Exit(1)
+}
+
+func main() {
+	var cfg config
+	if err := stepconf.Parse(&cfg); err != nil {
+		failf("Issue with input: %s", err)
+	}
+	stepconf.Print(cfg)
+
+	additionalParams, err := shellquote.Split(cfg.AdditionalParams)
+	if err != nil {
+		failf("Failed to parse additional parameters, error: %s", err)
+	}
+
+	fmt.Println()
+	log.Infof("Running test")
+
+	testCmd := command.New("flutter", append([]string{"test"}, additionalParams...)...).
+		SetStdout(os.Stdout).
+		SetStderr(os.Stderr)
+
+	fmt.Println()
+	log.Donef("$ %s", testCmd.PrintableCommandArgs())
+	fmt.Println()
+
+	if err := testCmd.Run(); err != nil {
+		failf("Running command failed, error: %s", err)
+	}
```

Comment:
are not there any test result we could export?

Reply:
couldn't find anything that worth to be exported unfortunately